src/DI/ContainerBuilder.php

```diff
@@ -149,6 +149,16 @@ public function addAlias($alias, $service)
 
 
 	/**
+	 * Removes the specified alias.
+	 * @param $alias
```

Comment:
Yes, I am sorry. I did not realize this comment doesn't make any sense. Changed and squashed...